Smartsheet.Net.Standard/Entities/Cell.cs

```diff
@@ -50,11 +50,6 @@ public Cell Build(bool? strict)
 			this.Column = null;
 			this.Strict = strict;
 
-			if (this.Hyperlink != null)
-			{
```

Comment:
@rgdick11 I can't help but think this was added to mitigate an issue we were having such asssss.. I dunno the API getting angry that an extra property is passed in. What I'm saying is I'm not sure this won't break existing code if they update their SDK.